lib/main_common.pm

```diff
@@ -544,6 +544,11 @@ sub load_jeos_tests {
         return;
     }
     unless (get_var('LTP_COMMAND_FILE')) {
+        if (check_var('ARCH', 'aarch64') && is_opensuse()) {
+            # Enable jeos-firstboot, due to boo#1020019
+            load_boot_tests();
+            loadtest "jeos/prepare_firstboot";
```

Comment:
And you want to call `load_boot_tests` still afterwards?

Reply:
Yup, the test triggers a reboot